orca/2.1-a-first-look-at-a-neural-network.ipynb

```diff
@@ -0,0 +1,518 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "output_type": "execute_result",
+     "data": {
+      "text/plain": [
+       "'2.4.3'"
+      ]
+     },
+     "metadata": {},
+     "execution_count": 1
+    }
+   ],
+   "source": [
+    "import keras\n",
+    "keras.__version__"
```

Comment:
Why using keras not tf.keras?

Reply:
The original notebook uses `keras` so I keep it here to minimize the changes, and it seems that `tf.keras` and `keras` function the same in this example. Do I need to change it?

Reply:
> The original notebook uses `keras` so I keep it here to minimize the changes, and it seems that `tf.keras` and `keras` function the same in this example. Do I need to change it?

Our recommendation is to use tf.keras; just add

```
from tensorflow import keras
```

Reply:
I got it, thanks